src/zerg/Zerg.ts

```diff
@@ -511,6 +511,17 @@ export class Zerg {
 		setOverlord(this, newOverlord);
 	}
 
+	// TODO add retire/reassignment logic
+	// Eg. creep get repurposed, it gets recycled, etc
+	/**
+	 * When a zerg has no more use for it's current overlord, it will be retired.
+	 * For now, that means RIP
+	 */
+	retire() {
```

Comment:
do you use this anywhere? if not, remove it.

Reply:
Used a bunch in power mining. Goal is to have them be reused, recycled, etc.
I think a worthwhile addition, even if right now it's not doing much